packages/flutter/lib/src/widgets/image.dart

```diff
@@ -44,6 +45,15 @@ ImageConfiguration createLocalImageConfiguration(BuildContext context, { Size si
   );
 }
 
+/// Prefetches an image into the image cache.
```

Comment:
This documentation should be much more detailed. For example:

* Saying that doing this will mean that if the same image is later used by an [Image] or [BoxDecoration] or [FadeInImage], it will probably be loaded faster.
* Saying when the future returned by this method will resolve, especially in the context of image providers that return multiple images (such as animated GIFs, once we support those)
* linking to ImageCache, Image, FadeInImage, with a brief description for why each may be relevant.
* A description of what the build context and size are used for.
* A mention that the image provider doesn't have to be the exact same instance reused later, so long as the image provider ends up pointing to the same key (which is what the image cache cares about).